### ISSUE TITLE
Performance drop if set --autoscale-lr 

### ISSUE THREAD

**Issue by Dchenhj** [OpenDriveLab/LaneSegNet#12]
Hi, I conducted two reproduction experiments on the same openlane_v2 dataset. Both experiments were trained using 8 NVIDIA Tesla V100 GPUs with a total batch size of 8, over the course of 24 training epochs. The only difference was whether the --autoscale_lr was set or not. I found that when the --autoscale_lr was enabled, there was a significant drop in performance.

From the code, whether to set this parameter does not affect the learning rate，So I'm confused about this.
<img width="711" alt="image" src="https://github.com/OpenDriveLab/LaneSegNet/assets/53369638/a24ec7b0-9187-4e09-a9ac-b02d405e22e6">

reproduced result with --autoscale_lr 
{'mAP': 0.13650985062122345, 'AP_ls': 0.16275863, 'AP_ped': 0.11026106, 'TOP_lsls': 0.10861310987314955}
reproduced result without --autoscale_lr 
{'mAP': 0.3187885582447052, 'AP_ls': 0.3224528, 'AP_ped': 0.3151243, 'TOP_lsls': 0.25731517483348604}

**Comment by sephyli** [comment]
I apologize for the inconvenience. I have identified the bug and will work to resolve it as soon as possible.

**Comment by sephyli** [comment]
This bug comes from the behavior mismatch between mmdet and mmdet3d. Thank you for bringing this to my attention.

Now the bug is fixed. Please pull the latest [commit](https://github.com/OpenDriveLab/LaneSegNet/commit/569b628cf9d63ec4713a253249f2d55d7526d606). If everything goes right, you can see `The batch size match the base batch size: 8, will not scaling the LR (0.0002)` in your log.

Apologize again for any inconvenience.

**Comment by Dchenhj** [comment]
Thank you for your excellent work. I appreciate your prompt response and for resolving my confusion. I am a beginner in  deep learning, and I am very grateful for helping me overcome my difficulties.
